packages/Webkul/Customer/src/Http/Controllers/SessionController.php

```diff
@@ -81,7 +81,7 @@ public function create()
         //Event passed to prepare cart after login
         Event::dispatch('customer.after.login', request('email'));
 
-        return redirect()->intended(route($this->_config['redirect']));
+        return redirect('/customer/account/profile');
```

Comment:
You can reproduce this problem for yourself. 
1. Login as admin 
2. Log out of admin
3. Login as normal customer

Result: You will be redirected to the admin login page.